data/projects.yml

```diff
@@ -398,3 +398,29 @@ project_list:
     github_user:
       - hlj24
     note: "In celebration of the first ten years of the Wired! Lab at Duke University (now the Digital Art History & Visual Culture Research Lab), the staff and faculty created a born-digital report that not only shares the lab's work with the world but also demonstrates the possibilities for Quire and static site generators for digital publication. The lab's members are committed to using sustainable methods for digital publishing while also exploring emerging platforms."
+
+  - id: "marrow_report"
+    src: /screenshots/_marrow_report.jpg
+    alt: "Getty Foundation: Getty Marrow Internship Program Impact Report"
+    type: Report
+    publisher: Getty Foundation
+    title_info: "Getty Marrow Internship Program Impact Report"
+    pub_date: 2020-10-22
+    url: https://www.getty.edu/interns-2020/report/
+    customization_level: moderate
+    github_user:
+      - geealbers
+    note: "This 2020 report assesses the impact of nearly 30 years of the Getty Marrow Undergraduate internship program, a program dedicated to diversifying the staff of museums and visual arts organizations. This is the first year Quire has been used to present report from the Getty Foundation. Many elements of Quire were customized, including fonts, graphics, and layout, to match the brand identity of the internship program."
```

Comment:
Great start. The report just isn't annual, and I want to credit the designers, so suggested edit:

"This 2020 report assesses the impact of nearly 30 years of the Getty Marrow Undergraduate internship program, a program dedicated to diversifying the staff of museums and visual arts organizations in Los Angeles. Designed by [Picnic Design](https://www.picnicdesign.org/), the report has a number of customized visual elements, including fonts, graphics, and layout, to match Getty’s brand identity."